libs/component-library/src/lib/progress-bar/progressBar.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { FxBox, FxBoxProps } from '../box/box';
+import { configureEaseInOutLayoutAnimation } from '../utils/animations';
+
+export type FxProgressBarProps = {
+  progress: number;
+  width?: number;
+  total?: number;
+} & FxBoxProps;
+
+const FxProgressBar = ({
+  width,
```

Comment:
this `width` prop isn't actually being used. `onLayout` will override it in the state, and a `width` prop is never sent to the `FxBox` below.

Reply:
Ah, let me add it back in with an example of custom width

Reply:
Done:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/17250443/186211631-21c06591-6f74-4faa-90f4-f4d8de637b65.png">